sources/projects/base64-encode-decode/description.md

```diff
@@ -0,0 +1,195 @@
+<style>
+th:nth-child(3), th:nth-child(6), th:nth-child(9), td:nth-child(3), td:nth-child(6), td:nth-child(9) {
```

Comment:
Pretty slick way of styling the table! Does this styling carry over to the rest of the page (such as to a table in the description or elsewhere)? I don't think that matters too much, but something to think about.

Reply:
Yes, that would carry over to other tables on the page. Fortunately, that's the only one. If another table were added, I'd have to preface all of those with `table:first-child`.